content/en/blog/README.md

```diff
@@ -0,0 +1,62 @@
+# Istio blog
+
+The Istio blog is owned by the [Steering committee](GET LINK) and run by the [Editorial Team](#leadership).
+
+This section covers documentation, processes, and roles for the [Istio blog](https://istio.io/latest/blog/2020/).
+
+## Leadership 
+
+- **Editorial Lead:** [steering member company A], [steering member company B]
+- **Technical Editors:** [contributor], [contributor]
+- **Copy Editors:** [technical writer], [technical writer], [María Cruz](https://github.com/macruzbar)
+- **Blog Community Managers:**  [María Cruz](https://github.com/macruzbar), [independent contributor]
+
+## Contact
+
+- Slack: [#toc-steering-questions](https://istio.slack.com/archives/C018VK0T9JTT)
+
+## Submit a Post
+
+Anyone can write a blog post and submit it for review. Content highlighting commercial products is allowed, provided it calls out how the product integrates with Istio in a clear way. Blogposts highlighting commercial products will be labeled as such on the Istio blog. Please refer to the [blog guidelines](#blog-guidelines) for more guidance.
+
+To submit a blog post, follow the steps below.
+
+1. [Sign the Contributor License Agreements](https://github.com/istio/community/blob/master/CONTRIBUTING.md#contributor-license-agreements) if you have not yet done so.
+1. Familiarize yourself with the Markdown format for existing blog posts in the [docs repository](https://github.com/istio/istio.io/tree/master/content/en/blog/2020). 
+1. Write your blog post in a text editor of your choice.
+1. Choose at least one of the following tags for your blogpost: New releases, Announcements, Trainings, Events, Commercial products
+1. (Optional) If you need help with markdown, check out [StakEdit](https://stackedit.io/app#) or read [GitHub's formatting syntax](https://help.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax) for more information. 
+1. Choose a directory in the [docs repository](ADD A LINK), and click **Create new file**.
```

Comment:
can we add a bullet for diagrams?  follow istio diagram guideline when possible: https://istio.io/latest/about/contribute/diagrams/